xml/planning-architecture-example_configurations.xml

```diff
@@ -68,6 +68,24 @@
     </para>
    </listitem>
   </itemizedlist>
+
+  <para>
+   &o_iron; multi-tenancy feature uses &o_netw; to manage the tenant
+   networks. The interaction between &o_netw; and the physical switch is
+   facilitated by &o_netw;'s Modular Layer 2 (ML2) plugin. The &o_netw; ML2
+   plugin supports drivers to interact with various networks, as each vendor
+   may have their own extensions. Those drivers are referred to as &o_netw; ML2
+   mechanism drivers, or simply mechanism drivers.
```

Comment:
maybe add emphases around *ML2 mechanism drivers* and *mechanism drivers*.

Reply:
Wird gemacht!